### DISCUSSION TITLE
[App Support]: Error when try to modify nvram boot-args

### DISCUSSION THREAD

**Discussion by Nikoz132** [PlayCover/PlayCover#476] [Q&A]
### App Store link

https://ipa.decrypt.day/app/id1517783697/

### Current issues

- [ ] App fails to install
- [ ] App fails to open
- [ ] App crashes randomly during usage
- [ ] App experiences graphical issues or broken UI
- [ ] App fails to connect to servers
- [X] Other

### More details

Can't sign in to the game, troubleshoot using the [documentation](https://docs.playcover.io/getting_started/troubleshoot_login.html#troubleshoot-app-login-issues), 

On step 2, when paste in `sudo nvram boot-args="amfi_get_out_of_my_way=0x1 ipc_control_port_options=0"` into the terminal, I was asked to enter the password, after that I get the error below,

Error setting variable - 'boot-args': (iokit/common) not permitted.

### Crash log

```shell
nikozhang@NikodeMacBook-Air ~ % sudo nvram boot-args="amfi_get_out_of_my_way=0x1 ipc_control_port_options=0"
Password:
Error setting variable - 'boot-args': (iokit/common) not permitted.
```

### Issue Language

- [X] Yes my issue is written in English

**Comment by amirsaam** [comment]
You are trying to put nvram boot-args while your SIP is not disabled.
That's why Terminal threw `not permitted` error.
